app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/TextAppearance.AppCompat.Headline"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:gravity="center"
-    android:text="@string/splash_fragment_title" />
\ No newline at end of file
+    android:background="@color/purple">
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:src="@drawable/splash_background_image"
+        tools:ignore="ContentDescription" />
+
+    <ImageView
+        android:layout_width="@dimen/scooter_image_width"
```

Comment:
I think you exported the wrong scooter image. You should be able to set width and height to wrap_content and `android:layout_gravity="center|start"` and it would be enough to make it look good.
Also with the right asset it won't be necessary to set negative margins

Reply:
![image](https://user-images.githubusercontent.com/84084708/185028827-878a7878-ae85-42d2-b199-c1fda64116a9.png)
Export this one. I think the asset in the style guide isn't formatted correctly